src/browser/constructLayoutEngine.js

```diff
@@ -382,6 +382,22 @@ function recurseRoutes({
           applicationElement = document.createElement("div");
           applicationElement.id = htmlId;
         }
+        if (typeof route.classNames === "string") {
```

Comment:
This should be called `className` or `class` rather than `classNames`. `class` is the html attribute name and `className` is the js property name. I think I lean more towards `className` since it's the javascript way of specifying it and this part of the code is javascript

Reply:
Totally agree with you here, will adapt